cmd/api/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+)
+
+var Version string
+
+type config struct {
+	port int
+	env  string
+}
+type application struct {
+	config config
+	// TODO: Add logger
+}
+
+var cfg config
+
+// Read value of command-line flags
+func init() {
+	flag.IntVar(&cfg.port, "port", 8080, "API server port")
```

Comment:
https://github.com/spf13/viper maybe?

Reply:
viper is good

Reply:
Is it ok If I using [godotenv](https://github.com/joho/godotenv) to load values from .env instead of using flags?